cl/search/models.py

```diff
@@ -2870,6 +2883,31 @@ class Meta:
         verbose_name_plural = "Opinion parentheticals"
 
 
+class ParentheticalGroup(models.Model):
+    opinion = models.ForeignKey(
+        Opinion, related_name="parenthetical_groups", on_delete=models.CASCADE
+    )
+    representative = models.ForeignKey(
+        Parenthetical,
+        related_name="represented_group",
+        on_delete=models.CASCADE,
+    )
+    score = models.FloatField(
+        db_index=True,
```

Comment:
Sorry, if my comment about semgrep wasn't clear. Can you move db_index into the `Meta.indexes` attribute, please?